arkane/ess/gaussian.py

```diff
@@ -57,8 +57,47 @@ class GaussianLog(ESSAdapter):
     arrays. GaussianLog is an adapter for the abstract class ESSAdapter.
     """
 
-    def __init__(self, path):
+    def __init__(self, path, check_for_errors=True):
         self.path = path
+        if check_for_errors:
+            self.check_for_errors()
```

Comment:
Since all `XXX Log` Classes share this same `__init__` function, can we simply modify the `__init__` for the parent class `ESSAdapter` ?

Reply:
That's a good suggestion. I pushed a change